frontend/src/pages/NewShoot.vue

```diff
@@ -179,7 +185,7 @@ export default {
         this.hibernationScheduleValid
     },
     isShootContentDirty () {
-      return !isEqual(this.newShootResource, this.shootResourceFromUIComponents())
+      return !isEqual(this.initialNewShootResource, this.shootResourceFromUIComponents())
```

Comment:
move to store

Reply:
do not move to store as currently newshootresource is only updated on page change and therefore store cannot tell if dirty